gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -152,7 +153,7 @@
  * throw or to return false.
  */
 public abstract class GoogleHadoopFileSystemBase extends FileSystem
-    implements FileSystemDescriptor {
+    implements FileSystemDescriptor, IOStatisticsSource {
```

Comment:
Do we need this ? 
Also please remove all the cosmetic changes in the file

Reply:
Removed